docs/api-standards.md

```
@@ -0,0 +1,745 @@
+    message: 'Request validation failed';
+    details: ValidationError[];
+  };
+  // ... meta
+};
+```
+
+## Route Naming Conventions
+
+### URL Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)